modules/nf-core/interproscan/main.nf

```diff
@@ -9,7 +9,7 @@ process INTERPROSCAN {
 
     input:
     tuple val(meta), path(fasta)
-    val(out_ext)
+    path(interproscan_database, stageAs: 'data')
```

Comment:
meta for the database just in case?

Reply:
It's always the same database for Interproscan. It's tied to the version and build of the software. Which databases inside that folder are used are toggled by the `-appl` flag.